### PR TITLE
Install elemental-operator from Rancher Markeplace chart

### LINKED ISSUE #139: Make the UI extension to install the charts from the market place
The idea is to make elemental installation a single step from the UI perspective.

### PR DESCRIPTION
**Pull request by aalves08**
Fixes #139 

- Use `v-clean-html` instead of `v-html`
- Bump shell version used to `0.3.27`
- Breakdown of `pkg/elemental/pages/index.vue` main view into components such as `DashboardView` and `InstallView`
- Create `InstallView` which enables user to install Elemental Operator from Rancher Marketplace

**To test:**
- You will need to have Rancher backend version `2.8.0` (2.8-head, 2.8.0-rc3, etc)
- Make sure you don't have already installed elemental operator
- Run this branch and go to elemental extension and follow the steps

**Preview:**
https://github.com/rancher/elemental-ui/assets/97888974/6b8781de-80ea-4b92-8d82-14994869ea8c

### CODE REVIEW

**Comment by aalves08** [comment]
FYI @fgiudici @juadk @davidcassany 

@juadk this will change the e2e tests. Added a preview for you guys to check how it will now work.

**Comment by juadk** [comment]
Looks good but one remark because I'm working on this, is it possible to deploy the dev operator version if I use the dev ui?
I mean, I can easily test installing stable operator but the purpose of the CI is to test the latest dev operator.
Without speaking about the UI extension, I'm trying to host the dev operator locally to expose it and be able to add a custom repository in the Rancher apps.

**Comment by aalves08** [comment]
> Looks good but one remark because I'm working on this, is it possible to deploy the dev operator version if I use the dev ui? I mean, I can easily test installing stable operator but the purpose of the CI is to test the latest dev operator. Without speaking about the UI extension, I'm trying to host the dev operator locally to expose it and be able to add a custom repository in the Rancher apps.

I'll have to do a bit of digging to see what's the best way to achieve it, but I think it's doable @juadk . 

Question:
- Do you test a built version of the `elemental-ui` or do you "run" the project locally when doing the e2e tests @juadk ?

File: pkg/elemental/pages/index.vue [hunk 1]
```diff
@@ -1,42 +1,28 @@
...
+import { ELEMENTAL_SCHEMA_IDS } from '../config/elemental-types';
 
 export default {
-  name:       'Dashboard',
+  name:       'BaseView',
   components: {
-    Loading, Banner, PercentageBar, ResourceTable, BuildIso
+    Loading,
+    InstallView,
+    // InstallView,
```
Comment (jordojordo) [inline pkg/elemental/pages/index.vue:19]: These commented lines for `InstallView` can be removed.

**Review by jordojordo — APPROVED** [review]
lgtm - a few commented lines that can be removed but everything works as expected.

**Comment by aalves08** [comment]
@jordojordo one final review, please 🙏  I really want a double-check here, as I've seen a bit of weirdness in my system (which disappeared after I uninstalled and re-installed again both the elemental-ui and operator). Thanks

File: pkg/elemental/pages/index.vue [hunk 2]
```diff
@@ -1,594 +1,58 @@
...
-      const allDispatches = await allHash(requests);
-
-      this.resourcesData = {};
-
-      this.resourcesData[ELEMENTAL_SCHEMA_IDS.MACHINE_REGISTRATIONS] = allDispatches.machineRegistrations;
-      this.resourcesData[ELEMENTAL_SCHEMA_IDS.MACHINE_INVENTORIES] = allDispatches.machineInventories;
-      this.resourcesData[this.ELEMENTAL_CLUSTERS] = filterForElementalClusters(allDispatches.rancherClusters);
-      this.resourcesData[ELEMENTAL_SCHEMA_IDS.MANAGED_OS_IMAGES] = allDispatches.managedOsImages;
-      this.resourcesData[ELEMENTAL_SCHEMA_IDS.MACHINE_INV_SELECTOR] = allDispatches.machineInvSelector;
+      const elementalSchema = this.$store.getters['management/schemaFor'](ELEMENTAL_SCHEMA_IDS.MACHINE_INVENTORIES)
```
Comment (jordojordo) [inline pkg/elemental/pages/index.vue:24]: Will the schemas always be populated in the store at this point?

Comment (aalves08) [reply pkg/elemental/pages/index.vue:24]: yes they will, as soon as the operator has been installed

**Review by jordojordo — APPROVED** [review]

**Pull request merged**